src/liballoc/boxed.rs

```diff
@@ -816,6 +816,26 @@ impl From<Box<str>> for Box<[u8]> {
     }
 }
 
+#[stable(feature = "box_from_array", since = "1.44.0")]
+impl<T, const N: usize> From<[T; N]> for Box<[T]>
+where
+    [T; N]: LengthAtMost32,
+{
+    /// Converts a `[T; N]` into a `Box<[T]>`
+    ///
+    /// This conversion allocates on the heap
+    /// and performs a copy of `slice`.
```

Comment:
"copy" can be easily misunderstood. What about "This conversion moves the array to newly heap-allocated memory" or something like that? So "move" instead of "copy". Also, `slice` should be called `array`.

Reply:
Okay, I will update the message.